src/schnetpack/representation/painn.py

```diff
@@ -135,10 +137,12 @@ def __init__(
         radial_basis: nn.Module,
         cutoff_fn: Optional[Callable] = None,
         activation: Optional[Callable] = F.silu,
-        max_z: int = 100,
+        max_z: int = 87,
```

Comment:
using radioactive elements is gonna be a problem now ;)

Reply:
reverted in cd06b83